# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1424646777 18000
#      Sun Feb 22 18:12:57 2015 -0500
# Node ID 2e556954ced88745ca4f4b9047a42f4527499f52
# Parent  26fb4bfa4193d77e167de715a44b7d32922b23b0
bartlett: Reword error message for consistency

* bartlett.m: Reword error message for consistency with other functions.

diff --git a/scripts/signal/bartlett.m b/scripts/signal/bartlett.m
--- a/scripts/signal/bartlett.m
+++ b/scripts/signal/bartlett.m
@@ -31,17 +31,17 @@
 
 function c = bartlett (m)
 
   if (nargin != 1)
     print_usage ();
   endif
 
   if (! (isscalar (m) && (m == fix (m)) && (m > 0)))
-    error ("bartlett: M has to be an integer > 0");
+    error ("bartlett: M must be a positive integer");
   endif
 
   if (m == 1)
     c = 1;
   else
     m = m - 1;
     n = fix (m / 2);
     c = [2*(0:n)/m, 2-2*(n+1:m)/m]';
